multiview_object_recognizer_node: 
{
	cg_dist_for_clutter_factor: 0, 
	cg_dot_distance: 0.2,
	cg_max_taken: 2, 
	cg_max_time_for_cliques_computation: 100, 
	cg_ransac_threshold: 0.015, 
	cg_size: 0.015, 
	cg_size_thresh: 7, 
	chop_z: 2.0, 
	dataset_path: /media/Data/datasets/TUW/test_set,  
	distance_keypoints_get_discarded: 2.5e-05, 
	do_eval: true, 
	do_ourcvfh: false, 
	do_shot: false,  
	do_sift: true, 
	extension_mode: 1, 
	hv_clutter_regularizer: 5, 
	hv_color_sigma_ab: 0.5, 
	hv_color_sigma_l: 0.6, 
	hv_inlier_threshold: 0.015, 
	hv_occlusion_threshold: 0.01, 
	hv_optimizer_type: 0, 
	hv_radius_clutter: 0.03, 
	hv_regularizer: 3, 
	hv_resolution: 0.005, 
	icp_iterations: 8, 
	icp_type: 1,
	icp_voxel_size: 0.005, 
	max_vertices_in_graph: 4, 
	models_dir: /media/Data/datasets/TUW/models/, 
	opt_type: 0, 
	recognizer_structure_sift: /media/Data/datasets/TUW/training_data, 
	scene_to_scene: true, 
	sequence_name: set_00004,
	training_dir_shot: /media/Data/datasets/TUW/shot_trained/, 
	training_dir_sift: /media/Data/datasets/TUW/sift_trained/, 
	visualize_output: true
}

